Blockzilla/AppDelegate.swift

```diff
@@ -206,13 +206,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         splashView?.animateHidden(true, duration: 0.25)
         if let url = queuedUrl {
-            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.openedFromExtension, object: TelemetryEventObject.app)
+            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.open, object: TelemetryEventObject.app, value: "extension")
 
             browserViewController.ensureBrowsingMode()
             browserViewController.submit(url: url)
             queuedUrl = nil
         } else if let text = queuedString {
-            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.openedFromExtension, object: TelemetryEventObject.app)
+            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.open, object: TelemetryEventObject.app, value: "extension")
```

Comment:
If this is an old probe, we can't just change the way we track it without cost - our data stream will have been interrupted.

Reply:
so do we want to keep it the way it was? I can update it back to what it was here and on the doc

Reply:
Yes, don't change old probes - just reflect them in the doc.